.claude/skills/skills-integrate-mcp-with-copilot/SKILL.md

```
@@ -0,0 +1,77 @@
+  // my-function.test.ts
+  import { myFunction } from './my-function';
+
+  describe('myFunction', () => {
+    it('should return expected result', () => {
+      expect(myFunction()).toBe('expected');
+    });
+  });
+  ```
+
+## Commands
+| Command           | Purpose                                      |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)